src/main/resources/db/migration/V2022005051100__create_table_departments.sql

```diff
@@ -0,0 +1,54 @@
+CREATE TABLE departments
+(
+    id          NUMBER(8) PRIMARY KEY,
+    name        VARCHAR2(30) UNIQUE NOT NULL,
```

Comment:
why is name UNIQUE, can be same department name in diff locations ?

Reply:
Yes, you are right, but if the is not unique we can have more departments with the same name in one location which does not have much sense. It should be unique per location.

Reply:
make unique name + location